fulp_modules/main_features/species/beefmen/mob/beefman.dm

```diff
@@ -722,3 +723,34 @@
 /datum/sprite_accessory/beef/mouth/smile2
 	name = "Smile2"
 	icon_state = "smile2"
+
+/obj/item/organ/tongue/beefman_tongue
+	name = "tongue"
```

Comment:
```suggestion
	name = "meaty tongue"
```

Reply:
OK ASSHOLE how do i change the code and edit the PR look here and listen carefully this is my second time using github ok?